azure-kit-demo/kubernetes.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: azure-kit-demo
+  namespace: default
```

Comment:
I would remove the namespace from the deployment definition, so it can be defined when applying the configuration

Reply:
Good point, done!